### ISSUE TITLE
crypto library 

### ISSUE THREAD

**Issue by remyut** [keystone-enclave/keystone#271]
Hi,

What crytpo lib do you recommend for ECDSA 256 p1 to run inside the enclave?
I can only find libsodium, but it only support eddsa (ed25519) 

Thanks

**Comment by dayeol** [comment]
Yes, thank you for bringing this up. Our ed25519 was modified and is non-standard. Rohit was working on switching to standard (and more common) SHA256 + ECDSA (https://github.com/keystone-enclave/keystone/tree/dev-rohit-sha256) for the measured boot + attestation, but I don't know how far he got.

**Comment by dayeol** [comment]
FYI, our measured boot is based on MIT Sanctum's secure bootloader (https://github.com/ilebedev/secure_bootloader/)

**Comment by asyarifstudio** [comment]
Thanks for the clarification @dayeol 

just wondering why you need to implement modified ED25519. why not use standard library like libsodium, libgcrypt or other lib? I think it's better to use existing and well-tested library, especially crypto to ensure the security for the crypto computation.
